custom/icds/data_management/migrations/0002_auto_20200220_0940.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nitpick: would be nice to give this file a human-readable name.

Reply:
this is already deployed on staging as well. I think django will pick it up well when creating new migrations so going to leave it as-is and not worry about modifying staging db to rollback/update etc.

Reply:
makes sense